pom.xml

```diff
@@ -63,12 +63,12 @@
 			<dependency>
 				<groupId>edu.isi.vista</groupId>
 				<artifactId>common-core-open</artifactId>
-				<version>8.2.1-SNAPSHOT</version>
+				<version>8.2.2-SNAPSHOT</version>
```

Comment:
Try depending on `8.2.1`.

Reply:
Couldn't get it to work with 8.2.2. IntelliJ couldn't find it

Reply:
Does it work for 8.2.1?

Reply:
It looks like changes since 8.2.1 pertain only to `IndexFlatGigaword`: https://github.com/isi-vista/nlp-util/commits/master

Reply:
Oops! I couldn't get it to work for 8.2.**1** but it works for 8.2.**2** so I think we should keep these changes.

Reply:
8.2.2 here as well, and below